tajo-plan/src/main/java/org/apache/tajo/plan/algebra/BaseAlgebraVisitor.java

```diff
@@ -52,221 +52,221 @@ public RESULT visit(CONTEXT ctx, Stack<Expr> stack, Expr expr) throws TajoExcept
-      break;
-
-    // String Operator or Pattern Matching Predicates
-    case LikePredicate:
-      current = visitLikePredicate(ctx, stack, (PatternMatchPredicate) expr);
-      break;
-    case SimilarToPredicate:
-      current = visitSimilarToPredicate(ctx, stack, (PatternMatchPredicate) expr);
-      break;
-    case Regexp:
-      current = visitRegexpPredicate(ctx, stack, (PatternMatchPredicate) expr);
-      break;
-    case Concatenate:
-      current = visitConcatenate(ctx, stack, (BinaryOperator) expr);
-      break;
-
-    // Arithmetic Operators
-    case Plus:
-      current = visitPlus(ctx, stack, (BinaryOperator) expr);
-      break;
-    case Minus:
-      current = visitMinus(ctx, stack, (BinaryOperator) expr);
-      break;
-    case Multiply:
-      current = visitMultiply(ctx, stack, (BinaryOperator) expr);
-      break;
-    case Divide:
-      current = visitDivide(ctx, stack, (BinaryOperator) expr);
-      break;
-    case Modular:
-      current = visitModular(ctx, stack, (BinaryOperator) expr);
-      break;
-
-    // Other Expressions
-    case Sign:
-      current = visitSign(ctx, stack, (SignedExpr) expr);
-      break;
-    case Column:
-      current = visitColumnReference(ctx, stack, (ColumnReferenceExpr) expr);
-      break;
-    case Target:
-      current = visitTargetExpr(ctx, stack, (NamedExpr) expr);
-      break;
-    case Function:
-      current = visitFunction(ctx, stack, (FunctionExpr) expr);
-      break;
-    case Asterisk:
-      current = visitQualifiedAsterisk(ctx, stack, (QualifiedAsteriskExpr) expr);
-      break;
-
-
-    case CountRowsFunction:
-      current = visitCountRowsFunction(ctx, stack, (CountRowsFunctionExpr) expr);
-      break;
-    case GeneralSetFunction:
-      current = visitGeneralSetFunction(ctx, stack, (GeneralSetFunctionExpr) expr);
-      break;
-    case WindowFunction:
-      current = visitWindowFunction(ctx, stack, (WindowFunctionExpr) expr);
-      break;
-
-
-    case DataType:
-      current = visitDataType(ctx, stack, (DataTypeExpr) expr);
-      break;
-    case Cast:
-      current = visitCastExpr(ctx, stack, (CastExpr) expr);
-      break;
-    case Literal:
-      current = visitLiteral(ctx, stack, (LiteralValue) expr);
-      break;
-    case NullLiteral:
-      current = visitNullLiteral(ctx, stack, (NullLiteral) expr);
-      break;
-    case DateLiteral:
-      current = visitDateLiteral(ctx, stack, (DateLiteral) expr);
-      break;
-    case TimeLiteral:
-      current = visitTimeLiteral(ctx, stack, (TimeLiteral) expr);
-      break;
-    case TimestampLiteral:
-      current = visitTimestampLiteral(ctx, stack, (TimestampLiteral) expr);
-      break;
-    case IntervalLiteral:
-      current = visitIntervalLiteral(ctx, stack, (IntervalLiteral) expr);
-      break;
-
-    default:
-      throw new TajoInternalError("Cannot support this type algebra \"" + expr.getType() + "\"");
+      case SetSession:
```

Comment:
The wrong coding format seems to be applied. {{switch}} and {{case}} should be the same column position.